airflow/providers/cncf/kubernetes/triggers/wait_container.py

```diff
@@ -0,0 +1,163 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
May be it is better to name this file as kubernetes_pod.py to follow naming convention in Airflow?

Reply:
So the names are driven by what the things do.  This is a trigger that waits until container done.  It is used by KPO now, but it doesn't mean that something else couldn't use it.

then, think about when you are reading KPO code
if you see "now let's defer to wait container trigger" you know what it is doing
if you see "now let's defer to kubernetes pod operator trigger", then you don't know quite as well what it's doing